doc/quickstart/configure.rst

```diff
@@ -479,58 +470,157 @@ speed with which the data can be read from and/or written to disk.
 
 .. _config-dask-default-scheduler:
 
-Dask default scheduler configuration
-------------------------------------
+Custom Dask default scheduler configuration
+-------------------------------------------
 
 The Dask default scheduler can be a good choice for recipes using a small
 amount of data or when running a recipe where not all preprocessor functions
-are lazy yet (see :issue:`674` for the current status). To use the the Dask
-default scheduler, comment out or remove all content of ``~/.esmvaltool/dask.yml``.
+are lazy yet (see :issue:`674` for the current status).
 
 To avoid running out of memory, it is important to set the number of workers
-(threads) used by Dask to run its computations to a reasonable number. By
-default the number of CPU cores in the machine will be used, but this may be
-too many on shared machines or laptops with a large number of CPU cores
+(threads) used by Dask to run its computations to a reasonable number.
+By default, the number of CPU cores in the machine will be used, but this may
+be too many on shared machines or laptops with a large number of CPU cores
 compared to the amount of memory they have available.
 
-Typically, Dask requires about 2GB of RAM per worker, but this may be more
+Typically, Dask requires about 2 GiB of RAM per worker, but this may be more
 depending on the computation.
 
-To set the number of workers used by the Dask default scheduler, create a file
-called ``~/.config/dask/dask.yml`` and add the following
-content:
+To set the number of workers used by the (thread-based) Dask default scheduler,
```

Comment:
```suggestion
To set the number of workers used by the threaded scheduler,
```

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2616/commits/f6fc37cf23c291066bc2f6fc6015860856c83c3c